man/man1/p11sak.1.in

```diff
@@ -111,7 +111,7 @@ attribute of the key and
 can be used to set the binary attributes of the key (see below for detailed description of the attributes).
 .B \-\-custom\-attr | \-c 
 .IR [[pub_attr]:[prv_attr]]
-can be used to set the binary attributes of the key (see below for deteiled describtion ans syntax).
+can be used to set the binary and CK_ULONG attributes of the key (see below for deteiled describtion ans syntax).
```

Comment:
see below for deteiled describtion -> see below for detailed description

Reply:
if we name it `CK_ULONG attributes` here, then the boolean attributes should probably be named `CK_BBOOL attributes` already in the previous patch to be consitent.

Reply:
We should be careful and think about how we will support non Boolean , non-ULONG attributes at a later time.  Just to avoid that we decide something now that hinders us from supporting other attribute types later on.

Reply:
Done